### DISCUSSION TITLE
Welcome to surge Discussions!

### DISCUSSION THREAD

**Discussion by sintaxi** [sintaxi/surge#433] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fu-sen** [comment]
Hi Brock. I'm from Japan. I already know and test various web services, but my website and blog are still managed by Surge.\
**The big advantage of Surge is that there is no limit on the number of sites and bandwidth.**\
Also, since the build environment [Harp](http://harpjs.com/) is isolated, Surge is a deployment-only service.\
I use [Hugo](https://gohugo.io/) etc, but it can be built locally. So Surge was ideal:

```
hugo
surge public
```

From our point of view, Surge does not seem to change for several years.\
But I know you're active lately. It will be revealed soon. 

Surge is alive. Surge will continue to be used in the future.

**Reply by sintaxi** [reply]
Hey, thanks for your interest and your help getting these discussions started! Indeed there has been a lot of activity in the background with surge. Many new features are about to released that will be documented here.

**Reply by fu-sen** [reply]
This discussion hasn't received any response other than us yet, but you'll be able to get it by doing something to Surge.
